config/locales/en.yml

```diff
@@ -23,6 +23,8 @@ en:
   submit: &submit Submit
   delete: &delete Delete
   remove: &remove Remove
+  cancel: &cancel Cancel
+  upload: &upload
```

Comment:
should this be `upload: *upload`  ?  

Or are you defining it here for the very first time?  In which case it should be `upload: &upload Upload`